weblate/addons/forms.py

```diff
@@ -200,7 +200,7 @@ class DiscoveryForm(BaseAddonForm):
         initial='po',
         required=True,
         help_text=_(
-            'Automatic detection might fail for some formats '
+            'Automatic detection might fail for some formats, '
             'and is slightly slower.'
```

Comment:
Hmm, there is no automatic detection anymore, so this needs more than grammar fixes. I'll adjust this one...